README.md

```diff
@@ -112,12 +112,15 @@ Contact: firstname.lastname@hpi.de
 -	Michael  Janke
 -	Max      Jendruk
 -	Marvin   Keller
+- Mirko    Krause
 -	Sven     Lehmann
+- Tom      Lichtenstein
 -	Jan      Mattfeld
 -	Arne     Mayer
```

Comment:
(random line)
with the changes from this PR, `PredicateCondition::Between` itself becomes ambiguous and always needs the two "inclusion-bools" to convey a precise meaning. Thus still having `PredicateCondition::Between` makes us prone to bugs. Should we have `PredicateCondition::Between`, `PredicateCondition::BetweenLowerExclusive`, `PredicateCondition::BetweenUpperExclusive` and `PredicateCondition::BetweenBothExclusive` @Bensk1

Reply:
Should the other PredicateConditions be added? SQL only supports the double-inclusive meaning, anyway.

Reply:
Yes, I think @Bensk1 agreed they should be added as well. And just to be clear: `PredicateCondition` is an entirely different level than SQL.

Reply:
Thanks for the explanation. I will add it soon.

I have to talked with Jan and assume that PredicateCondition::Between should be renamed to BetweenBothInclusive, too. Right?

Reply:
BetweenInclusive would do it for me, but no strong opinions

Reply:
Can somebody list the final decision here? I am confused by those `both/neither/left/right/exclusive`s. :)

Reply:
I want to turn it into: BetweenInclusive, BetweenLowerExclusive, BetweenUpperExclusive, BetweenExclusive

Reply:
and i agree with that

Reply:
Please have a look at it again